lib/galaxy/config/config_manage.py

```diff
@@ -80,7 +103,7 @@ def lint(self, args, app_desc, key, value):
 class _DeprecatedAndDroppedAction(_OptionAction):
     def converted(self, args, app_desc, key, value):
         print(f"Option [{key}] has been deprecated and dropped. It is not included in converted configuration.")
-        return DROP_OPTION_VALUE
+        return key, DROP_OPTION_VALUE
```

Comment:
Why do we need the `key` returned? (same in `_PasteAppFactoryAction`) We already have the value when we call the `converted` method, and that method does not modify the key. Am I missing something?

Reply:
It's to have consistency of return type for the `converted()` method across all subclasses of `_OptionAction`, see that the `_RenameAction` class below needs to return both key and value because it modifies the key and not the value.

Reply:
Oh, I see now. Thanks!